en/components/general/ignite-ui-licensing.md

```diff
@@ -63,21 +62,42 @@ This will allow you to seamlessly use a mix of packages from the public npm regi
 >[!NOTE]
 > `npm` is disallowing the use of the `"@"` symbol inside your username as it is considered as being "not safe for the net". Because your username is actually the email that you use for your Infragistics account it always contains the symbol `"@"`. That's why you must escape this limitation by replacing the `"@"` symbol with `"!!"` (two exclamation marks). For example, if your username is `"username@example.com"` when asked about your username you should provide the following input: `"username!!example.com"`.
 
-#### Now, to log in to our private feed using npm, run the adduser command and specify a user account and password:
+### Now, to log in to our private feed using npm
 
-```cmd
-npm adduser --registry=https://packages.infragistics.com/npm/js-licensed/ --scope=@infragistics --always-auth
-```
->[!NOTE]
-> If you are using `npm` version 9.0.0 or higher you need to set `--auth-type=legacy`
+#### npm version 9.4+
+<pre style="background:#141414;color:white;display:inline-block;padding:16x;margin-top:10px;font-family:'Consolas';border-radius:5px;width:100%">
+npm config set @infragistics:registry https://packages.infragistics.com/npm/js-licensed/
 
-```cmd
-npm adduser --registry=https://packages.infragistics.com/npm/js-licensed/ --scope=@infragistics --always-auth --auth-type=legacy
-```
+npm config set //packages.infragistics.com/npm/js-licensed/:username=${username}
+
+npm config set //packages.infragistics.com/npm/js-licensed/:email=${igemail}
+
+npm config set //packages.infragistics.com/npm/js-licensed/:_auth=${IG_AUTH_TOKEN}
+</pre>
+
+You can generate [Access Token](#access-token-usage) through your Infragistics profile.
+
+This approach is applicable to all prior versions of `npm`.
 
 > [!IMPORTANT]
 > If your account is not licensed (you are still using a Trial account) the private package feed won't be accessible to you e.g. it will return 404 or 403 error message. **Only licensed accounts can access the packages.infragistics private feed.**
 
+#### npm v9.0.x - 9.3.x
```

Comment:
@kdinev, @damyanpetev and I just tested `npm addUser` with npm 9.3.1 and this command is not applicable. It still returns an error. So about your first comment, should we still add the `auth-type=legacy`, I dont think there is a point of adding it. Lets remove the whole `v9.0.x - 9.3.x` section 

![image](https://user-images.githubusercontent.com/11193764/223103026-939e6013-119a-45d1-9932-85d76280e3d8.png)

Reply:
Agree.

Reply:
Aye, the legacy flag is irrelevant if the commands don't work in 9.0-9.3 anyway and might just have a 9+ section. 
On that note: Should we also add an explanation? Like:
"Our private feed doesn't currently support login/adduser commands with npm v9, so we recommend the following steps instead to add the required auth fields to the config:"

Reply:
As a matter of fact, perhaps the general token guidance should stay as the preferred approach even, regardless of version?